src/index.js

```diff
@@ -29,6 +29,6 @@ serviceWorker.register({
     if (registration && registration.waiting) {
       registration.waiting.postMessage({ type: 'SKIP_WAITING' });
     }
-    window.location.reload();
+    // window.location.reload();
```

Comment:
can we just remove this line instead of commenting? if commenting is really necessary, can we add a line to explain why its commented?